Stripe/STPCard.m

```diff
@@ -135,6 +137,54 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+#pragma mark - Description
+
+- (NSString *)description {
+    NSString *fundingDescription;
+
+    switch (self.funding) {
```

Comment:
- I agree about being more strict
- Ah yeah, good question. I think we should do it on a case-by-case basis. For source type, it definitely makes sense to have an unknown case, since source types are added pretty frequently. In this case, we should probably not add a new unknown case publicly, since we don't expect the enum to change. Maybe we can make a private sentinel value to represent an unknown enum value across the SDK? 
- Definitely agree that the current pattern isn't that elegant, and we can probably come up with something better, open to alternatives!